x/transfer/handler_test.go

```diff
@@ -0,0 +1,311 @@
+package transfer_test
+
```

Comment:
There is a bit of commented-out code in this test file. I'd recommend removing it if it's not intended to be re-enabled.

Reply:
yeah, there are a few places with that code commented. We can do that cleanup in #5558